packages/openapi-v3-types/src/openapi-v3-spec-types.ts

```diff
@@ -0,0 +1,401 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/openapi-v3-types
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+/*
+ * OpenApiSpec - A typescript representation of OpenApi 3.0.0
+ */
+
+// tslint:disable:max-line-length
+
+import * as OAS3 from 'openapi3-ts';
+// Export spec interfaces from the community module if missing in our package
+export * from 'openapi3-ts';
+
+export type OpenApiSpec = OAS3.OpenAPIObject;
+/**
+ * Custom extensions can use arbitrary type as the value,
+ * e.g. a string, an object or an array.
+ */
+// tslint:disable-next-line:no-any
+export type ExtensionValue = any;
+
+/**
+ * The location of a parameter.
+ * Possible values are "query", "header", "path" or "cookie".
+ * Specification:
+ * https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md
```

Comment:
A more specific link would be `https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#parameter-locations` here

Reply:
@biniam, good catch, i put the whole link and specify a manual search by `search "The location of the parameter"` below because I didn't notice there is an anchor for it lol, swagger 2 has one but I thought oas3 doesn't.